source/common/ssl/context_impl.h

```diff
@@ -68,6 +68,14 @@ class ContextImpl : public virtual Context {
    */
   static bool dNSNameMatch(const std::string& dnsName, const char* pattern);
 
+  /**
+   * Determines whether the given URI matches 'pattern' which may end with a wildcard.
+   * @param uriPattern the pattern to match against (foo.bar/baz/ *)
```

Comment:
Please remove space and perhaps replace this with the same example as in docs (i.e. `foo.com/*`).